config/jobs/kubernetes/wg-k8s-infra/trusted/sig-contribex-k8s-triage-robot.yaml

```diff
@@ -1,30 +1,122 @@
+        is:pr
+        base:master
+        label:kind/api-change
+        -label:do-not-merge/work-in-progress
+        -label:do-not-merge/hold
+        -label:needs-rebase
+        -label:api-review
+        -project:kubernetes/13
+        NOT "complete the pre-review checklist and request an API review"
+      - --updated=5m
+      - --token=/etc/github-token/token
+      - |-
+        --comment=This PR [may require API review](https://git.k8s.io/community/sig-architecture/api-review-process.md#what-apis-need-to-be-reviewed).
+
+        If so, when the changes are ready, [complete the pre-review checklist and request an API review](https://git.k8s.io/community/sig-architecture/api-review-process.md#mechanics).
+
+        Status of requested reviews is tracked in the [API Review project](https://github.com/orgs/kubernetes/projects/13).
+      - --ceiling=10
+      - --confirm
+      volumeMounts:
+      - name: token
+        mountPath: /etc/github-token
+    volumes:
+    - name: token
+      secret:
+        secretName: k8s-triage-robot-github-token
+
+- name: ci-k8s-triage-robot-cla
   interval: 10m
   decorate: true
   cluster: k8s-infra-prow-build-trusted
   annotations:
     testgrid-dashboards: sig-contribex-k8s-triage-robot
     description: A sample job to make sure things work
-    testgrid-tab-name: api-review-help
+    testgrid-tab-name: cla
   spec:
     containers:
     - image: gcr.io/k8s-prow/commenter:v20210722-64d720b97e
       command:
       - /app/robots/commenter/app.binary
       args:
-      # a sample job to confirm the commenter and token work
       - |-
-        --query=is:issue
+        --query=org:kubernetes
+        org:kubernetes-sigs
+        org:kubernetes-client
+        org:kubernetes-csi
+        is:pr
         is:open
-        author:spiffxp
-        label:area/github-management
-        NOT "This is a test comment to confirm that k8s-triage-robot can comment on issues"
-      - --updated=5m
+        -label:"cncf-cla: no"
+        -label:"cncf-cla: yes"
+      - --token=/etc/github-token/token
+      - |-
+        --comment=Unknown CLA label state. Rechecking for CLA labels.
+
+        Send feedback to sig-contributor-experience at [kubernetes/community](https://github.com/kubernetes/community).
+        /check-cla
+      - --template
+      - --ceiling=10
+      - --confirm
+      volumeMounts:
+      - name: token
+        mountPath: /etc/github-token
+    volumes:
+    - name: token
+      secret:
+        secretName: k8s-triage-robot-github-token
+
+- name: ci-k8s-triage-robot-close
+  interval: 1h
+  decorate: true
+  annotations:
+    testgrid-dashboards: sig-contribex-k8s-triage-robot
+    description: Closes rotten issues after 30d of inactivity
+    testgrid-tab-name: close
+  spec:
+    containers:
+    - image: gcr.io/k8s-prow/commenter:v20210722-64d720b97e
+      command:
+      - /app/robots/commenter/app.binary
+      args:
+      - |-
+        --query=org:kubernetes
```

Comment:
I'm also good with this as-is but if we want to do a test run first, we could probably add a new job for triage-robot targeting only specific repos instead of the whole org.

Reply:
I'm willing to tackle whatever noise comes from this not working as expected.  I'm reasonably confident that https://github.com/kubernetes/test-infra/pull/22992 caught all of the issues related to commenting